source/fundamentals/data-formats/document-data-format-pojo.txt

```diff
@@ -18,11 +18,12 @@ MongoDB Java driver using plain old Java objects (`POJOs <https://en.wikipedia.o
 POJOs are often used for data encapsulation, separating business logic from
 data representation.
 
-In the following sections we show:
+In the following sections we show an example on how to configure the
+driver to serialize and deserialize POJOs and how to read and write to
+documents using POJOs. 
 
-- A POJO example
-- How to configure the driver to serialize and deserialize POJOs
-- Sample code to read and write documents to MongoDB using POJOs
+If you would like learn how to read and write to documents, see
+:ref:`java-read-operations` and :ref:`java-write-operations`.
```

Comment:
Issue: The descriptions of the links as well as the pages to which it directs (landing pages) lack information on what read and write operations are. 

Issue: The placement is also confusing since it's not clear to a reader that doesn't already know what the links contain differ from the aforementioned "how to read and write to documents using POJOs", or whether they're related.

Issue: This paragraph is duplicated on line 220.

Reply:
The duplication was intended since the ticket states to help guide the user to CRUD. From our discussion, I thought you wanted an earlier redirect in the case the user didn't intend to land on this page but somehow did.

I'll remove this redirect instance since I initially didn't think it made sense to have early on.

Reply:
I don't recall asking for an earlier mention, duplicate content, or redirect if a user didn't intend to land on the page.
Here's what I wanted to communicate about this ticket in any case:

- This ticket originated from user feedback regarding lack of content on CRUD operations this page.
- This is a page on working with POJOs that includes some information on insert and find operations using POJOs which contrasts with the rest of the CRUD documentation which uses Documents. As a reader, from this page, how would you want to be led in your learning journey?

If you have any questions, please reach out on Slack.